srcpkgs/jp2a/template

```diff
@@ -1,17 +1,18 @@
 # Template file for 'jp2a'
 pkgname=jp2a
-version=1.0.7
+version=1.1.1
 revision=1
 build_style=gnu-configure
-hostmakedepends="automake"
-makedepends="libjpeg-turbo-devel libcurl-devel"
+hostmakedepends="automake autoconf autoconf-archive pkg-config"
+makedepends="libjpeg-turbo-devel libpng-devel libcurl-devel ncurses-devel"
 short_desc="JPEG image to ASCII art converter"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="Subhaditya Nath <sn03.general@gmail.com>"
 license="GPL-2.0-only"
-homepage="https://github.com/cslarsen/jp2a"
-distfiles="https://github.com/cslarsen/jp2a/archive/v${version}.tar.gz"
-checksum=e509d8bbf9434afde5c342568b21d11831a61d9942ca8cb1633d4295b7bc5059
+homepage="https://github.com/Talinx/jp2a"
+distfiles="https://github.com/Talinx/jp2a/archive/v${version}.tar.gz"
```

Comment:
Can we just use the release tarball to avoid autoreconf?
```suggestion
distfiles="https://github.com/Talinx/jp2a/releases/download/v${version}/jp2a-${version}.tar.gz"
```